servers/graphql-kotlin-spring-server/src/main/resources/graphql-graphiql.html

```diff
@@ -0,0 +1,70 @@
+<!--
```

Comment:
retrieved it from
https://github.com/graphql/graphiql/tree/main/examples/graphiql-cdn